types/angular-material/index.d.ts

```diff
@@ -1,6 +1,11 @@
 // Type definitions for angular-material 1.1
 // Project: https://github.com/angular/material, https://material.angularjs.org
-// Definitions by: Blake Bigelow <https://github.com/blbigelow>, Peter Hajdu <https://github.com/PeterHajdu>, Davide Donadello <https://github.com/Dona278>, Geert Jansen <https://github.com/geertjansen>, Edward Knowles <https://github.com/eknowles>
+// Definitions by: Blake Bigelow <https://github.com/blbigelow>
```

Comment:
Note: all but line 8 (for the authors part) are formatting changes.
Line 8 is adding myself to authors.